LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Ferdi
+Copyright (c) 2022 Ferdium
```

Comment:
Let's keep this consistent indeed: Ferdium Contributors
Or should we go for Team? Because if we assign the copyright to Contributors, anyone with a single line of contribution will own the copyright? Not sure how this works legally.

Reply:
Done - changed to `Ferdium Contributors`

Reply:
AFAIK, technically everyone with a single line of contribution hold a part of the copyright. It's why [CLA](https://opensource.com/article/18/3/cla-vs-dco-whats-difference) are a thing, so this line won't really change something (disclaimer: I do not have any experience as a lawyer)